packages/babel-plugin-transform-nullish-coalescing-operator/README.md

```diff
@@ -0,0 +1,54 @@
+# @babel/plugin-transform-nullish-coalescing-operator
+
+> Replace `??` with an inline helper.
+
+## Example
+
+**In**
+
+```javascript
+var foo = object.foo ?? "default";
+```
+
+**Out**
+
+```javascript
+var _object$foo;
+
+var foo = (_object$foo = object.foo, _object$foo !== null && _object$foo !== void 0 ? _object$foo : "default");
+```
+
+> **NOTE:** We cannot use `!= null` here because `document.all` is `!= null` and
```

Comment:
Nit: '`document.all` is `!= null`' should be '`document.all` is `== null`'

Reply:
Fixed, thanks.